### DISCUSSION TITLE
adb連接一直失敗

### DISCUSSION THREAD

**Discussion by StarchP0tat0** [MaaAssistantArknights/MaaAssistantArknights#8919] [Q&A]
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/125759352/f0aebeb3-fb8c-4db0-8e51-2fdd54cc8c56)
我跟隨常見問題嘗試强制替換ADB
但是也是失敗了
重啓3次電腦也是沒有用
模式改了也沒用
想看看有沒有人能夠幫幫我

**Comment by ABA2396** [comment]
https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/new/choose
